### ISSUE TITLE
Porkbun dns resolver not working

### ISSUE THREAD

**Issue by blmhemu** [traefik/traefik#8201]
### Do you want to request a *feature* or report a *bug*?
Bug

### What did you do?
Configured TLS provider (porkbun)

### What did you expect to see?
A TLS certificate issued from letsencrypt / porkbun

### What did you see instead?
A TLS certificate issued from traefik

### Output of `traefik version`: (_What version of Traefik are you using?_)

docker `traefik:v2.4`

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

cli
```
"--certificatesresolvers.porkbunresolver.acme.email=REDACTED",
"--certificatesresolvers.porkbunresolver.acme.caserver=https://acme-staging-v02.api.letsencrypt.org/directory",
"--certificatesresolvers.porkbunresolver.acme.storage=acem.json",
"--certificatesresolvers.porkbunresolver.acme.dnschallenge=true",
"--certificatesresolvers.porkbunresolver.acme.dnschallenge.provider=porkbun",
"--certificatesresolvers.porkbunresolver.acme.dnschallenge.resolvers=REDACTED",
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
time="2021-06-21T10:24:34+05:30" level=error msg="Unable to obtain ACME certificate for domains \"REDACTED\" : cannot get ACME client unrecognized DNS provider: porkbun" providerName=porkbunresolver.acme
```

### Misc

Considering this as a bug (instead of feature) because the [docs](https://doc.traefik.io/traefik/https/acme/#dnschallenge) say porkbun is supported.

The porkbun provider is added in the latest `lego  v4.4.0` (https://github.com/go-acme/lego/releases/tag/v4.4.0)
Whereas traefik currently uses `lego v4.3.1` https://github.com/traefik/traefik/blob/2ccdc419d0139a892270654ab8d4b68aa323d342/go.mod#L33
Upgrading the version to 4.4.0 should probably close this issue.

If needed, I can create a minor PR (change version in gomod, run `go mod tidy`)

**Comment by rtribotte** [comment]
Hello @blmhemu,

Thanks for your interest in Traefik!

This is indeed fixed by the update to lego `v4.4.0`.
As this has already been addressed by #8179, I'll close this issue.
You should be able to use porkbun dns provider with the next bug fix release `v2.4.9`, which will be release in the coming weeks.

closed by #8179

**Comment by blmhemu** [comment]
Glad to hear it fixed in 2.4.9
Eagerly waiting for the new release.
